worker-testing-util/pom.xml

```diff
@@ -43,22 +43,22 @@
         <dependency>
             <groupId>com.github.workerframework</groupId>
             <artifactId>worker-configs</artifactId>
-            <version>2.0.0-SNAPSHOT</version>
```

Comment:
There's no need for the changes to this file, since the maven versions plugin updates the version numbers to keep them in-sync anyway.